deepchecks/core/serialization/check_failure/wandb.py

```diff
@@ -0,0 +1,65 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module containing Wandb serializer for the CheckFailuer type."""
+import typing as t
+
+from deepchecks.core import check_result as check_types
+from deepchecks.core.serialization.abc import WandbSerializer
+from deepchecks.core.serialization.common import prettify
+
+try:
+    import wandb
+    from wandb.sdk.data_types.base_types.wb_value import WBValue
```

Comment:
what is this?

Reply:
wandb is not a part of our default requirements, the user needs to install it separately if he wants to work with wandb serialization logic and we need to inform him about this as quickly as possible

Reply:
what is the WBValue?

Reply:
base type for all wandb data types - [docs](https://docs.wandb.ai/ref/python/data-types)